### ISSUE TITLE
Add 2x PNG sprites for the icons

### ISSUE THREAD

**Issue by addyosmani** [google/material-design-icons#2]
The current version of the icons only provide a 1 x resolution sprite. We'd ideally also ship a compatible 2 x version (retina).

Hang in tight if you're waiting on this. It'll come soonish :)

**Comment by RoelN** [comment]
Since spritesheets are not part of this repo/package anymore, I'm closing this issue.
